packages/frontend/src/hooks/useChartSummariesV2.ts

```diff
@@ -0,0 +1,30 @@
+import {
+    ContentType,
+    type ApiChartContentResponse,
+    type ApiError,
+} from '@lightdash/common';
+import { useQuery } from '@tanstack/react-query';
+import { API_PREFIX_V2, lightdashApi } from '../api';
+
+const getChartSummariesInProjectV2 = async (projectUuid: string) => {
+    return lightdashApi<ApiChartContentResponse['results']>({
+        apiPrefix: API_PREFIX_V2,
+        url: `/content?projectUuids=${projectUuid}&contentTypes=${ContentType.CHART}&pageSize=${Number.MAX_SAFE_INTEGER}`, // TODO: remove pageSize max once we have pagination
+        method: 'GET',
+        body: undefined,
+    });
+};
+
+export const useChartSummariesV2 = (projectUuid: string) => {
```

Comment:
I created a similar hook, mine was more generic, with URLparams generation util method, we can keep both for now I guess